dwave/embedding/chain_strength.py

```diff
@@ -37,10 +37,28 @@
 __all__ = ['uniform_torque_compensation', 'scaled']
 
 def uniform_torque_compensation(bqm, embedding=None, prefactor=1.414):
-    """Chain strength that attempts to compensate for torque that would break
-    the chain.
-
-    The RMS of the problem's quadratic biases is used for calculation.
+    r"""Chain strength that attempts to compensate for chain-breaking torque.
+
+    The root mean square (RMS) of the problem's quadratic biases is used to
+    calculate a value of chain strength that preforms reasonably well on many
+    problems.
+
+    As described in [Ray2020]_, optimal chain strength for a :term:`clique`
```

Comment:
An annealed wavefunction develops long range order as the transverse field is reduced. It is expected that for quantum dynamics to be efficient chain strength should be chosen so that chains become rigid in coordination with the development of this long range order [Ray2020]_ [https://journals.aps.org/prx/abstract/10.1103/PhysRevX.5.031040]. In hard frustrated models (such as spin-glasses) this dictates that the chain strength should grow in proportion to the typical variable connectivity and root mean square coupling strength.